### PR TITLE
Adding Information icon and dialog box

### PR DESCRIPTION
**Pull request by jongund**
* Adds information button next to rule and element summary information
* Button opens a dialog box with information on the abbreviations used in the summary information
* Updated some of the labels in AInspector views

### CODE REVIEW

**Comment by jongund** [comment]
@nhoyt 
Can you approve this pull request and then we can merge the code into main.

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +14,12 @@
 | 1.1.1        |   9/16/21  | Updated the extension to support ARIA 1.2 and ARIA in HTML requirements
 | 0.96.0       |  10/30/19  | Fixes bug in highlight option not being remembered
 
+
+### 2.2.0
+* Added information button to rule summary and element summary grids to provide documentation on the meaning of the symbols
+* Updating labeling of views to improve indemnification of information in each view
```
Comment (nhoyt) [inline CHANGELOG.md:20]: Not sure why the word 'indemnification' was used here. Would suggest: 'Updated labeling of views to provide more context for the information in each view'

File: README.md [hunk 2]
```diff
@@ -28,6 +28,10 @@
 
 ## Version History
 
+### 2.2.0
+* Added information button to rule summary and element summary grids to provide documentation on the meaning of the symbols
+* Updating labeling of views to improve indemnification of information in each view
```
Comment (nhoyt) [inline README.md:33]: See comment for CHANGELOG.md

File: src/_locales/en/messages.json [hunk 3]
```diff
@@ -274,11 +274,16 @@
     "description": "Label for hidden results"
   },
 
-  "notApplicablelabel": {
+  "notApplicableLabel": {
     "message": "Not Applicable",
     "description": "Label for not applicable results"
   },
 
+  "notApplicableDescRule": {
```
Comment (nhoyt) [inline src/_locales/en/messages.json:282]: It look like this is another message that was created for this branch, but is not being used.

File: src/_locales/en/messages.json [hunk 4]
```diff
@@ -1027,6 +1062,49 @@
   "shortcutAllreadyUsed": {
     "message": "The $key is already being used.",
     "description": "Feedback to use if the shortcut is already defined."
+  },
+
+  "summaryInfoButtonLabel": {
```
Comment (nhoyt) [inline src/_locales/en/messages.json:1067]: Is this message being used?

**Pull request merged**